src/WinUI.TableView.csproj

```diff
@@ -1,13 +1,27 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Uno.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
-    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
-    <WindowsSdkPackageVersion>10.0.22621.38</WindowsSdkPackageVersion>
+    <TargetFrameworks>
+      net8.0-desktop;
+      net8.0;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.26100;
+      net8.0-browserwasm;
+    </TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <UnoSingleProject>true</UnoSingleProject>
+    <OutputType>Library</OutputType>
+    <!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
     <Nullable>enable</Nullable>
-    <UseWinUI>true</UseWinUI>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DisableImplicitUnoPackages>true</DisableImplicitUnoPackages>
```

Comment:
Would you recall why you needed to add this line?

Reply:
I added this to remove unwanted dependencies. my main concern was Uno.WinUI in case of net8.0-windows.
![image](https://github.com/user-attachments/assets/988e4785-4880-47ee-b9e9-aa36240a09bd)

Reply:
Got it! Most of those dependencies are dev-time only, they won't appear in your packages.